app/auth/authV2/actions/AuthoriseAndRetrieveIndividual.scala

```diff
@@ -98,7 +98,7 @@ class AuthoriseAndRetrieveIndividual @Inject()(val authorisedFunctions: Frontend
       lazy val optMtdId: Option[String] =
         getValueFromEnrolment(Constants.mtdEnrolmentName, Constants.mtdEnrolmentIdentifierKey)
 
-      optMtdId.fold(throw new MissingMtdId) {
+      optMtdId.fold(throw InsufficientEnrolments("Missing MTDId")) {
```

Comment:
I guess this is Individual specific exception? so might be a good idea to specific this in the error message?